training_config/bidirectional_language_model.jsonnet

```diff
@@ -58,10 +58,11 @@ local BASE_ITERATOR = {
       //     "tokens": ["<S>", "</S>"],
       //     "token_characters": ["<>/S"]
       // },
```

Comment:
Please update the path to this file in https://github.com/allenai/allennlp/blob/master/tutorials/how_to/training_transformer_elmo.md. You should also grep the codebase for any other references to this file. And recall that this won't help external users who may have references to this file already. Please consider this carefully when weighing whether a rename should be performed.

Reply:
done, thanks for pointing that out